docs/api.rst

```diff
@@ -3188,11 +3189,13 @@ Search for assets(Levenshtein)
       }
 
    :reqjson int limit: This signifies the limit of records to return as per the `sql spec <https://www.sqlite.org/lang_select.html#limitoffset>`__.
-   :reqjson list[string] order_by_attributes: This is the list of attributes of the asset by which to order the results. By default we sort using ``name``.
-   :reqjson list[bool] ascending: Should the order be ascending? This is the default. If set to false, it will be on descending order.
    :reqjson string value: A string to be used to search the assets. Required.
-   :reqjson string[optional] evm_chain: A string representing the name of a supported EVM chain used to filter the result. e.g "ethereum", "optimism", "binance", etc.
-
+   :reqjson int[optional] chain_id: An integer representing the id of a supported EVM chain used to filter the result. e.g 1, 10, 57, etc.
```

Comment:
As per discord.


That aside there is a problem here. I thought that the api docs were just out of date as they tend to be
But here the frontend just gave the value they know. Which is "ethereum", "optimism" etc. and we turned it into ChainID in marshmallow. That was fine. In thatcase it could have stayed as is. Otherwise the frontend needs to also keep a mapping of chain id to blockchain which is unnecessary. So essentially undo this whole thing on the interface only.

Which mean evm_chain comes in from the frontend as a name but marshmallow transforms it into chainid as it did before.